lib/rubocop/cop/primer/deprecated_arguments.rb

```diff
@@ -33,6 +33,16 @@ class DeprecatedArguments < BaseCop
         # }
         #
         DEPRECATED = {
+          is_label_inline: nil,
+          with_icon: nil,
+          is_label_visible: {
+            false: "visually_hide_label: true",
```

Comment:
@jonrohan I think lint is failing on this syntax. Any ideas how to move forward?

Reply:
I think the lint check is warning you that `false:` with the special trailing colon hash syntax turns `false` into a Ruby symbol when you probably wanted a boolean.